test/testunusedvar.cpp

```diff
@@ -742,7 +742,7 @@ class TestUnusedVar : public TestFixture {
 
         functionVariableUsage("void foo()\n"
                               "{\n"
-                              "    int i = 0,code=10;\n"
```

Comment:
what warning did you get here?

I think some `unreadVariable` would be a FP. A `redundantInitialization` would be good!